test/extended/setup.sh

```diff
@@ -345,6 +345,10 @@ readonly EXCLUDED_TESTS=(
 
 	# this tests the _kube_ downgrade. we don't support that.
 	"\[Feature:Downgrade\]"
+
+	# upstream flakes:
```

Comment:
When you say "flake"... what does that mean? We aren't really in the habit of disabling tests that flake out once in a while. Why are these different?